### PR TITLE
fix startup bug in Firefox (fixes #1770)

### LINKED ISSUE #1770: Application not detected in Mozilla Firefox
**Describe the bug**
Mozilla Firefox Ember Inspector does not detect application

**To Reproduce**
1. Open Ember app in Mozilla Firefox browser with Ember Inspector installed
2. Open developer tools
3. Go to Ember tab

**Expected behavior**
Application is loaded, with component tree and other information

**Screenshots**
image

**Environment**
Ember: 3.17.1
Mozilla Firefox: 94.0 (64-bit) (MacOS)

**Additional context**
Chrome extension seems to be working fine; Might be related to Nov 1 update, since last week there didn't seem to be any issues

### PR DESCRIPTION
**Pull request by luxferresum**
## Description
Discussed this in Discord with @chancancode. There was a bug starting the inspector in FF. This fixes it, tho I'm not sure this is the right thing to do to fix it.

### CODE REVIEW

File: ember_debug/utils/rsvp.js [hunk 1]
```diff
@@ -5,10 +5,15 @@ let module, RSVP;
...
   RSVP = module.default;
+
+  // The RSVP module should have named exports for `Promise`, etc,
+  // but some old versions do not and provide `RSVP.Promise`, etc.
+  if (!('Promise' in module)) {
+    module = RSVP;
+  }
 } catch {
   module = RSVP = Ember.RSVP;
 }
```
Comment (chancancode) [inline ember_debug/utils/rsvp.js:16]:
```suggestion
}

```

**Review by chancancode — APPROVED** [review]
Did you say there was another spot like this that was causing issue for you? Should we apply a similar treatment there?

**Comment by mariojankovic** [comment]
Any ETA on this? I've termporarily switched to a chromium based browser until this gets fixed but very much like to continue using firefox instead for my day-to-day dev needs.

**Comment by rwwagner90** [comment]
@mariojankovic this has been in the firefox store for awhile now.

**Comment by mariojankovic** [comment]
@rwwagner90 awesome, thanks for the heads up!

**Pull request merged**